tools/ff_sfd_pseudoenc

```diff
@@ -0,0 +1,1153 @@
+10B80..10BAF; Psalter Pahlavi
+10C00..10C4F; Old Turkic
+10C80..10CFF; Old Hungarian
+10D00..10D3F; Hanifi Rohingya
+10E60..10E7F; Rumi Numeral Symbols
+10F00..10F2F; Old Sogdian
+10F30..10F6F; Sogdian
+11000..1107F; Brahmi
+11080..110CF; Kaithi
+110D0..110FF; Sora Sompeng
+11100..1114F; Chakma
+11150..1117F; Mahajani
+11180..111DF; Sharada
+111E0..111FF; Sinhala Archaic Numbers
+11200..1124F; Khojki
+11280..112AF; Multani
+112B0..112FF; Khudawadi
+11300..1137F; Grantha
+11400..1147F; Newa
+11480..114DF; Tirhuta
+11580..115FF; Siddham
+11600..1165F; Modi
+11660..1167F; Mongolian Supplement
+11680..116CF; Takri
+11700..1173F; Ahom
+11800..1184F; Dogra
+118A0..118FF; Warang Citi
+11A00..11A4F; Zanabazar Square
+11A50..11AAF; Soyombo
+11AC0..11AFF; Pau Cin Hau
+11C00..11C6F; Bhaiksuki
+11C70..11CBF; Marchen
+11D00..11D5F; Masaram Gondi
+11D60..11DAF; Gunjala Gondi
+11EE0..11EFF; Makasar
+12000..123FF; Cuneiform
+12400..1247F; Cuneiform Numbers and Punctuation
+12480..1254F; Early Dynastic Cuneiform
+13000..1342F; Egyptian Hieroglyphs
+14400..1467F; Anatolian Hieroglyphs
+16800..16A3F; Bamum Supplement
+16A40..16A6F; Mro
+16AD0..16AFF; Bassa Vah
+16B00..16B8F; Pahawh Hmong
+16E40..16E9F; Medefaidrin
+16F00..16F9F; Miao
+16FE0..16FFF; Ideographic Symbols and Punctuation
+17000..187FF; Tangut
+18800..18AFF; Tangut Components
+1B000..1B0FF; Kana Supplement
+1B100..1B12F; Kana Extended-A
+1B170..1B2FF; Nushu
+1BC00..1BC9F; Duployan
+1BCA0..1BCAF; Shorthand Format Controls
+1D000..1D0FF; Byzantine Musical Symbols
+1D100..1D1FF; Musical Symbols
+1D200..1D24F; Ancient Greek Musical Notation
+1D2E0..1D2FF; Mayan Numerals
+1D300..1D35F; Tai Xuan Jing Symbols
+1D360..1D37F; Counting Rod Numerals
+1D400..1D7FF; Mathematical Alphanumeric Symbols
+1D800..1DAAF; Sutton SignWriting
+1E000..1E02F; Glagolitic Supplement
+1E800..1E8DF; Mende Kikakui
+1E900..1E95F; Adlam
+1EC70..1ECBF; Indic Siyaq Numbers
+1EE00..1EEFF; Arabic Mathematical Alphabetic Symbols
+1F000..1F02F; Mahjong Tiles
+1F030..1F09F; Domino Tiles
+1F0A0..1F0FF; Playing Cards
+1F100..1F1FF; Enclosed Alphanumeric Supplement
+1F200..1F2FF; Enclosed Ideographic Supplement
+1F300..1F5FF; Miscellaneous Symbols and Pictographs
+1F600..1F64F; Emoticons
+1F650..1F67F; Ornamental Dingbats
+1F680..1F6FF; Transport and Map Symbols
+1F700..1F77F; Alchemical Symbols
+1F780..1F7FF; Geometric Shapes Extended
+1F800..1F8FF; Supplemental Arrows-C
+1F900..1F9FF; Supplemental Symbols and Pictographs
+1FA00..1FA6F; Chess Symbols
+20000..2A6DF; CJK Unified Ideographs Extension B
+2A700..2B73F; CJK Unified Ideographs Extension C
+2B740..2B81F; CJK Unified Ideographs Extension D
+2B820..2CEAF; CJK Unified Ideographs Extension E
+2CEB0..2EBEF; CJK Unified Ideographs Extension F
+2F800..2FA1F; CJK Compatibility Ideographs Supplement
+E0000..E007F; Tags
+E0100..E01EF; Variation Selectors Supplement
+F0000..FFFFF; Supplementary Private Use Area-A
+100000..10FFFF; Supplementary Private Use Area-B"""
```

Comment:
`fontTools.unicodedata` should have some code to help here instead of embedding the Unicode blocks data.

Reply:
The tension here is between fewer lines in the script and a dependency. Right now the script's only dependency is on Python 3 and its standard library. That has certain advantages for less sophisticated/integrated projects.

Reply:
We depend on FontTools already for other scripts, so I’d rather use it than maintain duplicated code that needs to be updated with future Unicode releases. The less code I have to maintain, the simpler my life is.

Reply:
In that case, would you prefer to remove the code from the repository entirely and add a dependency on the script? I already have its repository set up as a python package ( https://github.com/skef/fontforge_coverage_tools) and it's easy enough to make it available via PyPI. (I was holding off on that until I had a bit more experience with it.) 

I'm not envisioning this as something maintained by you (as the Libertinus maintainer) or any other individual project, but that would be maintained via pull requests to `fontforge_coverage_tools`. I just figured it would be easier on everyone to "distribute" the script via font repos rather than require individual contributors to add a new python package on their local machines.

Reply:
I'm now realizing I was never explicit about this:

1. Although I wrote this tool based on my experience working with Libertinus, I designed as a general-purpose script that can be used by any FontForge-based project.
2. It's `home` is the `fontforge_coverage_tools` repository, and any changes should be made via push request there.
3. It may eventually be used primarily as a locally-installed python package, but I would prefer to get more experience with it before "advertising" it that way.
4. For a variety of reasons (including unsophisticated contributors) many projects may want to use it in the way it is set up in Libertinus now, as an independent script that can copied from `fontforge_coverage_tools` as needed.
5. Projects distributing the script in this way benefit from it having no dependencies, which is why I took the block-quote approach rather adding a dependency on another package.